lib/galaxy/tool_util/deps/container_resolvers/__init__.py

```diff
@@ -4,11 +4,18 @@
     abstractmethod,
     abstractproperty,
 )
+from typing import (
+    Optional,
+    TYPE_CHECKING,
+)
 
 
 from galaxy.util.bunch import Bunch
 from galaxy.util.dictifiable import Dictifiable
 
+if TYPE_CHECKING:
+    from beaker.cache import Cache
```

Comment:
Won't that still break type checking in e.g cwltool ?

Reply:
We can install https://pypi.org/project/Beaker/ (yes?) for type checking, no problem.